example/run.bash

```
@@ -0,0 +1,100 @@
+# run
+make build
+./bin/manager \
+    -kubeconfig "$tmpdir/platform.kubeconfig" \
+    -source-kubeconfig "$consumers" \
+    -target-kubeconfig "$providers" \
+    -group example.platform-mesh.io \
+    -kind VM \
+    -version v1alpha1 \
+    &
+broker_pid=$!
+trap "kill $broker_pid; wait $broker_pid" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)